### PR TITLE
Chore: POK-5 Add Axios

### LINKED ISSUE #5: POK-5- 🧰 Tooling - chore: Add `axios`
# Add the `axios` HTTP client

## Install
```bash
npm install axios
```

### PR DESCRIPTION
**Pull request by mathiuskitchens**
Added `axios` HTTP client

[optional footer(s)]

# Description

Added Axios HTTP client

## Type of change

- [ ] - fix: - patches a bug in the codebase
- [ ] - feat: - introduces a new feature
- [ ] - build: - relates to build tooling / project scaffolding
- [x] - chore: - non-code related changes, like adding packages / libraries
- [ ] - docs: - changes related to documentation

## Related Issue

Does this change have a related issue in the project?

- [x] - Yes - Closes #5 
- [ ] - No

### CODE REVIEW

**Review by wijohnst — COMMENTED** [review]
Looks like you have some merge conflicts in `package.json` to fix.

**Comment by mathiuskitchens** [comment]
Let me know if this conflict got resolved the way I did it. It's been a few days and I didn't quite remember.

File: package.json [hunk 1]
```diff
@@ -24,5 +24,8 @@
     "eslint": "^8.33.0",
     "parcel": "^2.8.3",
     "prettier": "2.8.3"
+  },
+  "dependencies": {
+    "axios": "^1.3.2"
```
Comment (wijohnst) [inline package.json:29]: Yep, that looks right to me.

**Review by wijohnst — APPROVED** [review]
LGTM!

**Pull request merged**